### PR TITLE
Add code to inject Google Analytics script.

### LINKED ISSUE #138: Google Analytics should be part of all apps
It would be best to add the GA code during deployment. 

cc @nicolaskruchten

### PR DESCRIPTION
**Pull request by shammamah**
Closes #138. 

## About 
Added an environment variable in CircleCI with the name `DASH_GA_CODE` that contains the Google Analytics code, and modified the predeploy script to inject that code in an `app.index_string` just above `app.layout` for the app being deployed.  

The script was added into the default `app.index_string` (https://github.com/plotly/dash/blob/7b23e7960f8788fe9050dc2008c99bcc765d0571/dash/dash.py#L40).

### CODE REVIEW

**Comment by shammamah** [comment]
@rpkyle I just added this in for the Python deployments -- I think we can do something similar with R, but obviously with some of the variable names (like `app.layout`) changed. Let me know what you think!

File: predeploy.py [hunk 1]
```diff
@@ -21,5 +21,23 @@
...
+    lines = []
+    ga_line = 0
+    with open(os.path.join(app_path, "app.py"), "r") as f:
+        lines = f.readlines()
+
+    try:
+        ga_line = [i for i in range(len(lines)) if "app.layout" in lines[i]]
+        lines.insert(
+            ga_line[0],
+            "app.index_string = '''\n{}\n'''\n\n".format(os.environ["DASH_GA_CODE"]),
```
Comment (nicolaskruchten) [inline predeploy.py]: this is likely not what you want to do, as it will result in no HTML being in `index_string`...

File: predeploy.py [hunk 2]
```diff
@@ -21,5 +21,23 @@
 else:
     for f in pyfiles:
         shutil.copyfile(os.path.join(app_path, f), f)
+    lines = []
+    ga_line = 0
+    with open(os.path.join(app_path, "app.py"), "r") as f:
+        lines = f.readlines()
```
Comment (nicolaskruchten) [inline predeploy.py]: you should be able to move the next block into here and go line by line without slurping it all into memory, by using the iterator.

File: apps/dash-tsne/assets/plotly_ga.css [hunk 3]
```diff
@@ -1,7 +0,0 @@
-(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```
Comment (nicolaskruchten) [inline apps/dash-tsne/assets/plotly_ga.css]: what is this doing in a CSS file??? :)

Comment (shammamah) [reply apps/dash-tsne/assets/plotly_ga.css]: No idea! 😕

**Pull request merged**